### ISSUE TITLE
Module build failed (from ./node_modules/babel-loader/lib/index.js):  

### ISSUE THREAD

**Issue by jnarowski** [nuxt/create-nuxt-app#725]
Node 
`14.15.5`

? Project name: microsaascollective
? Programming language: JavaScript
? Package manager: Yarn
? UI framework: Tailwind CSS
? Nuxt.js modules: Axios - Promise based HTTP client
? Linting tools: ESLint, Prettier
? Testing framework: Jest
? Rendering mode: Universal (SSR / SSG)
? Deployment target: Server (Node.js hosting)
? Development tools: jsconfig.json (Recommended for VS Code if you're not using typescript)
? Continuous integration: None
? Version control system: Git

I am creating a fresh Nuxt 3.5.2 app with npx and a new node install (14.15.5).

When I try to run `yarn dev` I get this error:

```jnarowski@Onyx microsaascollective % yarn dev
yarn run v1.22.10
$ nuxt

   ╭───────────────────────────────────────────────────────╮
   │                                                       │
   │   Nuxt @ v2.15.0                                      │
   │                                                       │
   │   ▸ Environment: development                          │
   │   ▸ Rendering:   server-side                          │
   │   ▸ Target:      server                               │
   │                                                       │
   │   Listening: http://localhost:3000/                   │
   │                                                       │
   │   Tailwind Viewer: http://localhost:3000/_tailwind/   │
   │                                                       │
   ╰───────────────────────────────────────────────────────╯

ℹ Preparing project for development                                                                                                                    08:30:11
ℹ Initial build may take a while                                                                                                                       08:30:11
ℹ Discovered Components: .nuxt/components/readme.md                                                                                                    08:30:11
✔ Builder initialized                                                                                                                                  08:30:11
✔ Nuxt files generated                                                                                                                                 08:30:11

● Client █████████████████████████ building (11%) 9/9 modules 0 active


✖ Server
  Compiled with some errors in 385.37ms


✖ Client
  Compiled with some errors in 535.76ms

✖ Server
  Compiled with some errors in 385.37ms


 ERROR  Failed to compile with 1 errors                                                                                                friendly-errors 08:30:12


 ERROR  in ./.nuxt/client.js                                                                                                           friendly-errors 08:30:12

Module build failed (from ./node_modules/babel-loader/lib/index.js):                                                                   friendly-errors 08:30:12
Error: [BABEL] /Users/jnarowski/Dev/sourceborn/src/microsaascollective/.nuxt/client.js: Cannot find module '/Users/jnarowski/Dev/sourceborn/src/microsaascollective/node_modules/@babel/runtime/helpers/interopRequireWildcard.js' (While processing: "/Users/jnarowski/Dev/sourceborn/src/microsaascollective/node_modules/@nuxt/babel-preset-app/src/index.js")
    at createEsmNotFoundErr (internal/modules/cjs/loader.js:907:15)
    at finalizeEsmResolution (internal/modules/cjs/loader.js:900:15)
    at resolveExports (internal/modules/cjs/loader.js:432:14)
    at Function.Module._findPath (internal/modules/cjs/loader.js:472:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:867:27)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at n (/Users/jnarowski/Dev/sourceborn/src/microsaascollective/node_modules/jiti/dist/v8cache.js:2:2364)
    at Object.<anonymous> (/Users/jnarowski/Dev/sourceborn/src/microsaascollective/node_modules/regenerator-transform/lib/visit.js:9:31)
    at Module.o._compile (/Users/jnarowski/Dev/sourceborn/src/microsaascollective/node_modules/jiti/dist/v8cache.js:2:2670)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at n (/Users/jnarowski/Dev/sourceborn/src/microsaascollective/node_modules/jiti/dist/v8cache.js:2:2364)
    at Object.<anonymous> (/Users/jnarowski/Dev/sourceborn/src/microsaascollective/node_modules/regenerator-transform/lib/index.js:6:14)
                                                                                                                                       friendly-errors 08:30:12
 @ multi ./node_modules/eventsource-polyfill/dist/browserify-eventsource.js (webpack)-hot-middleware/client.js?reload=true&timeout=30000&ansiColors=&overlayStyles=&path=%2F__webpack_hmr%2Fclient&name=client ./.nuxt/client.js
                                                                                                                                       friendly-errors 08:30:12
ℹ Waiting for file changes                                                                                                                             08:30:12
ℹ Memory usage: 117 MB (RSS: 206 MB)                                                                                                                   08:30:12
ℹ Listening on: http://localhost:3000/
```

**Comment by finnbuster** [comment]
I've tried this on multiple machines and am getting the same issue. I can't tell if it's an issue with Nuxt or create-nuxt-app

**Comment by Andugaa** [comment]
Hi, i have the same error. 
![nuxtError](https://user-images.githubusercontent.com/66117939/108794428-041cdd00-7586-11eb-8428-a62b23f7494d.png)

**Comment by oiseedev** [comment]
I got the same error! Re-install @babel/preset-env is useless.

**Comment by danielroe** [comment]
Likely duplicate of https://github.com/nuxt/nuxt.js/issues/8882 - see there for workaround.

**Comment by pi0** [comment]
Thanks for report. Issue fixed in [2.15.2](https://github.com/nuxt/nuxt.js/releases/tag/v2.15.2)

**Comment by Mosavi87** [comment]
you can run this comment for this issue

yarn add @babel/plugin-proposal-private-property-in-object --dev
